2-mandatory/2-currency-code-lookup.js

```diff
@@ -18,9 +18,16 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
 function createLookup(countryCurrencyCodes) {
-  // write code here
+  let countryCurrencyObject={}
+  for (let i=0; i<countryCurrencyCodes.length; i++){
+  let newarr= countryCurrencyCodes[i]
+
+  countryCurrencyObject[`${newarr[0]}`]=newarr[1];
```

Comment:
This line is works however`${newarr[0]}` can be newarr[0]. Template literals are used to print variables inside strings. We don't need to have a string so can avoid using a template literal in this context.